src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -207,7 +207,7 @@ LRESULT IslandWindow::_OnSizing(const WPARAM wParam, const LPARAM lParam)
     // bad parameters, which we won't have, so no big deal.
     LOG_IF_FAILED(GetDpiForMonitor(hmon, MDT_EFFECTIVE_DPI, &dpix, &dpiy));
 
-    const long minWidthScaled = minimumWidth * dpix / USER_DEFAULT_SCREEN_DPI;
+    const LONG minWidthScaled = minimumWidth * dpix / USER_DEFAULT_SCREEN_DPI;

```

Comment:
Weird, this one didn't turn into an `auto`?

Reply:
No, the expression evaluates to an `UINT` actually, because `dpix` is an `UINT`.
The old code unsafely casts it to a `long`, but I think in practice that's just fine. It's unlikely there to be over 2B pixels wide screens after all. 😅

I'll revert this change to `long` to reduce the diff by 1.